novawallet/Modules/NetworkManagement/CustomNetwork/CustomNetworkViewController.swift

```diff
@@ -193,6 +193,12 @@ private extension CustomNetworkViewController {
             for: .touchUpInside
         )
 
+        rootView.urlInput.pasteButton.addTarget(
```

Comment:
If we use `TextWithServiceInputView`'s `editingChanged`, it will send autocomplete requests every time the user tries to manually enter or edit the address. Most of the time, the address will be invalid, resulting in alerts to the user. I think it's better to use the `pasteButton` to trigger the action in that case.